src/main/org/tvrenamer/controller/util/FileUtilities.java

```diff
@@ -128,6 +128,16 @@ public static boolean areSameDisk(Path pathA, Path pathB) {
         }
     }
 
+    public static boolean isSameFile(final Path path1, final Path path2) {
```

Comment:
As you know, I wrote a lot of new stuff, and am trying to check it in, in an understandable way.  It's possible I added this method to FileUtilities before I used it, but as of the current sources, it is used in org.tvrenamer.controller.MoveRunner.  And the method in FileUtilities does do the catch, and always returns a boolean.